### PR TITLE
Add a guide to Build a private Custom API in Python and Flask

### PR DESCRIPTION
**Pull request by mrinalwadhwa**
Hi Snowflake team 👋 

The pull request a new quick start guide to [Build a **private** Custom API in Python and Flask using Ockam](https://github.com/mrinalwadhwa/sfquickstarts/blob/mrinal/add-private-api-guide/site/sfguides/src/build_a_private_custom_api_in_python/build_a_private_custom_api_in_python.md).

The new guide is very similar to the existing guide - [Build a Custom API in Python and Flask.](https://github.com/Snowflake-Labs/sfquickstarts/blob/master/site/sfguides/src/build_a_custom_api_in_python/build_a_custom_api_in_python.md)

Looking forward to your thoughts and feedback.

### CODE REVIEW

File: site/sfguides/src/build_a_private_custom_api_in_python/build_a_private_custom_api_in_python.md [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+
+DROP ROLE IF EXISTS DATA_API_ROLE;
+DROP DATABASE IF EXISTS API;
+DROP INTEGRATION IF EXISTS OCKAM;
+DROP COMPUTE POOL IF EXISTS API_POOL;
+DROP WAREHOUSE IF EXISTS DATA_API_WH;
+```
+
+<!-- ------------------------ -->
+## Conclusion
```
Comment (iamontheinet) [inline site/sfguides/src/build_a_private_custom_api_in_python/build_a_private_custom_api_in_python.md]: @mrinalwadhwa Pl rename this step to **Conclusion And Resources** and also provide links to relevant resources. For example, https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html#7

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/build_a_private_custom_api_in_python/build_a_private_custom_api_in_python.md (hunk 1 above)
Comment (mrinalwadhwa) [reply site/sfguides/src/build_a_private_custom_api_in_python/build_a_private_custom_api_in_python.md]: @iamontheinet great suggestion.
Just pushed an updated version with a `Conclusion and Resources` section https://github.com/Snowflake-Labs/sfquickstarts/pull/1521/commits/d9c0eff225ec8d66c2b9738218273f8291309208

**Comment by mrinalwadhwa** [comment]
@iamontheinet thank you for merging! Is there some other step I should do to get this published on? https://quickstarts.snowflake.com

**Comment by iamontheinet** [comment]
@mrinalwadhwa it's been deployed -- https://quickstarts.snowflake.com/guide/build_a_private_custom_api_in_python/index.html

**Comment by mrinalwadhwa** [comment]
Awesome, thank you!

**Pull request merged**